### PR TITLE
Migrate String Refs to Functional ones - MM 16039

### PR DESCRIPTION
**Pull request by Spotts9**
#### Summary
Migrate String Refs to functional ones within `./components/adminconsole/remove_file_setting.jsx`

Fixes https://github.com/mattermost/mattermost-server/issues/16039

https://mattermost.atlassian.net/browse/MM-29856

### CODE REVIEW

**Review by mkraft — APPROVED** [review]

File: components/admin_console/remove_file_setting.jsx [hunk 1]
```diff
@@ -22,12 +21,18 @@ export default class RemoveFileSetting extends Setting {
         };
     }
 
+    constructor(props) {
+        super(props);
+
+        this.remove_button = React.createRef();
```
Comment (deanwhillier) [inline components/admin_console/remove_file_setting.jsx:27]: Two things regarding naming here:
1.  We prefer to use lower camel case for variables names
2. Typically we append 'Ref' to the end of these variable names

Could we please get the name of this variable changed to something like `removeButtonRef`?

```suggestion
        this.removeButtonRef = React.createRef();
```

**Review by deanwhillier — CHANGES_REQUESTED** [review]
@Spotts9, thanks so much for the contribution, much appreciated!

One item to consider below when you have a minute! 🙂

**Review by deanwhillier — APPROVED** [review]
LGTM, thanks @Spotts9!

**Review by stevemudie — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
Thank you for the contribution @Spotts9

**Pull request merged**